apps/fluent-tester/src/FluentTester/TestComponents/Common/Slider.tsx

```diff
@@ -121,7 +121,7 @@ export const Slider: React.FunctionComponent<ISliderProps> = (props: ISliderProp
         setThumbLocation(initialThumbLocation);
```

Comment:
It may have been one of those things to shut up typescript. The "!" after a variable tells typescript "I know this /can/ be undefined but I am telling you that we guarantee a valid value here." The type of minimum/maxium/initialValue is probably `umber | undefined` so TS would complain here without the "!". "!" doesn't actually enforce any value so if it/is/ undefined we'd probably crash or do something weird instead.

Reply:
(Probably the more "proper" way is to instantiate a new variable of type number. We shouldn't be directly modifying props anyway.)